packages/zol-json/src/PGJson.ts

```diff
@@ -59,6 +59,15 @@ export class PGJson {
         return Unsafe.unsafeFunN("jsonb_build_object", args, PGJson.parser);
     }
 
+    /**
+     * check if JSON array contains text
+     *
+     * This is the PostgreSQL `?` operator
+     */
+    public static arrayContainsText<s>(lhs: Col<s, PGJson>, rhs: Col<s, string>): Col<s, boolean | null> {
```

Comment:
I think the `?` operator will never return NULL when given two valid (non-null) arguments, so the return value of this function should be `Col<s, boolean>`